packages/core/template/project/SettingUp.md

```diff
@@ -0,0 +1,66 @@
+---
+  layout: default.md
+  title: "Setting up and getting started"
```

Comment:
Page not accessible from the main site nav. Applies to several other pages - eg: logging. Please do not have these floating pages that are not accessible from site nav, it is unlikely users will be able to find them...

Reply:
For these pages, it is directly under their own sections for the DG so I think it is quite accessible since the site nav may not have direct links to them, but it brings them to the section which will clearly show the links to them. What do you think? Maybe it's better to overhaul the current format such that each page is standalone from the site nav?